code/modules/loadout/categories/glasses.dm

```diff
@@ -53,3 +53,19 @@
 /datum/loadout_item/glasses/eyepatch
 	name = "Eyepatch"
 	item_path = /obj/item/clothing/glasses/eyepatch
+
+/datum/loadout_item/glasses/geist_gazers
+	name = "Geist Gazers"
+	item_path = /obj/item/clothing/glasses/geist_gazers
+
+/datum/loadout_item/glasses/psych
+	name = "Psych Glasses"
+	item_path = /obj/item/clothing/glasses/psych
+
+/datum/loadout_item/glasses/kim
```

Comment:
Is this one prescription? If so it might be best to sort it up with the other prescription ones, keep them grouped together

Reply:
it gets scrambled anyways in the loadout select screen but ok